src/main/java/io/stargate/sgv2/jsonapi/api/exception/WebApplicationExceptionMapper.java

```diff
@@ -12,8 +15,23 @@
 public class WebApplicationExceptionMapper {
 
   @ServerExceptionMapper
-  public RestResponse<CommandResult> genericExceptionMapper(WebApplicationException e) {
-    Throwable toReport = null != e.getCause() ? e.getCause() : e;
+  public RestResponse<CommandResult> webApplicationExceptionMapper(WebApplicationException e) {
+    // 06-Nov-2023, tatu: Let's dig the innermost root cause; needed f.ex for [jsonapi#448]
+    //    to get to StreamConstraintsException
+    Throwable toReport = e;
+    while (toReport.getCause() != null) {
```

Comment:
Used to only dig into immediate root cause, but some cases have at least 2 levels of nesting (StreamConstraintsException -> JsonMappingException -> JsonApiException f.ex).